src/back/project/src/model/save_timer_serialize.cpp

```diff
@@ -27,7 +27,7 @@ SaveTimer Parse(
 	const auto projectId = projectIdStr.empty() ? boost::uuids::uuid{} : utils::BoostUuidFromString(projectIdStr);
 
 	return {
-		.id = json["id"].As<int>(),
+		.id = json["id"].As<int64_t>(),
```

Comment:
Давай проинициализируем все поля в структуре чтобы компилятор не выдавал предупреждения
```suggestion
		.spaceId = {},
```

это нужно сделать для всех моделей

Reply:
Готово (для всех моделей).